layers/+readers/epub/packages.el

```diff
@@ -23,6 +23,8 @@
       (kbd "L") 'nov-next-document
       (kbd "[") 'nov-previous-document
       (kbd "]") 'nov-next-document
+      (kbd "d") 'nov-scroll-up
+      (kbd "u") 'nov-scroll-down
```

Comment:
It seems like these keybindings are the other way around.

Reply:
It's because `nov-scroll-up` is actually `down` in Spacemacs conventions (following the `pdf` layer).

`nov-scroll-up` is like `C-d` with `evil`; `nov-scroll-down` is like `C-u`.

Reply:
Thanks for clarifying that.